src/components/Launch/LaunchForm/LaunchRoleInput.tsx

```diff
@@ -109,7 +109,9 @@ export const LaunchRoleInputImpl: React.RefForwardingComponent<
                     </Typography>
                 </div>
                 <TextField
-                    id={`launchform-role-${AuthRoleStrings[AuthRoleTypes.IAM]}`}
+                    id={`launchform-role-${
+                        AuthRoleStrings[AuthRoleTypes.IAM].inputLabel
+                    }`}
```

Comment:
I think we should eliminate whitespaces from the `id` field.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id#:~:text=id%20's%20value%20must%20not,have%20one%20single%20ID%20value

Reply:
Fixed.